### PR TITLE
introduction of a basic scene, and gameobject definition used by the scene

### PR DESCRIPTION
**Pull request by co0p**
* loadingImage is now fading in
* dots of loadingText are now animated

### CODE REVIEW

File: assets.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package tankism
```
Comment (co0p) [inline assets.go:1]: golang embed does not allow to import from parent (e.g ..) therefore we now have a asset graveyard file...

File: cmd/tankism/main.go [hunk 2]
```diff
@@ -1,18 +1,54 @@
...
 
 import (
+	_ "embed"
 	"github.com/co0p/tankism"
+	"github.com/co0p/tankism/cmd/tankism/start"
 	"github.com/hajimehoshi/ebiten/v2"
 	"log"
 )
 
+type Client struct {
```
Comment (co0p) [inline cmd/tankism/main.go:11]: moved client to cmd... this forces us to follow the import restrictions from golang. Golang does not allow import cycles. Therefore we follow the rule: 

The higher the folder the more it is allowed to import. 
Lower packages are not allowed to import from higher packages.

File: cmd/tankism/start/scene.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	scene.loadingText = tankism.NewLoadingText(&scene)
+
+	return &scene
+}
+
+func (s *StartScene) Init() error {
+	return nil
+}
+
+func (s *StartScene) Draw(screen *ebiten.Image) {
```
Comment (co0p) [inline cmd/tankism/start/scene.go:29]: delegating the draw and update to each gameobject

File: loadingImage.go [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	op.GeoM.Scale(scaleX, scaleY)
+	op.GeoM.Translate(x, y)
+	op.ColorM.ChangeHSV(1, 1, l.currentAlpha)
+
+	screen.Fill(colorBlack)
+	screen.DrawImage(l.image, op)
+}
+
+func (l *LoadingImage) Update() error {
+	if l.currentAlpha < ALPHA_MAX {
```
Comment (co0p) [inline loadingImage.go:57]: this allows us to have a basic fade in animation of the image

File: loadingText.go [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+	}
+}
+
+func (l *LoadingText) Draw(screen *ebiten.Image) {
+	w, h := l.scene.WindowDimension()
+	text.Draw(screen, l.text, l.font, w-300, h-100, l.primaryColor)
+}
+
+func (l *LoadingText) Update() error {
+	l.frameCount++
```
Comment (co0p) [inline loadingText.go:40]: this allows us to animate the text.

File: scene.go [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package tankism
+
+import "github.com/hajimehoshi/ebiten/v2"
+
+type Scene interface {
```
Comment (co0p) [inline scene.go:5]: initial draft. Def. not final.

**Review by jevbelikov — APPROVED** [review]

**Pull request merged**